### ISSUE TITLE
Lower time interval than a day

### ISSUE THREAD

**Issue by seb128** [powersj/ubuntu-bug-triage#18]
The topic was discussed on #15 , while the current day interval works fine for the server team daily job usecase for manual usage it would be nice to be able to specific a timestamp or an unit lower than day.

Copying from the previous report

'the case of a bug triager who manually use the tool to do a triaging round monday at 7am and then another one tuesday a 6pm and would like to review the bugs that changed in between those'

**Comment by powersj** [comment]
@seb128 thanks for filing this. For a range, it sounds like you are thinking a) down the hour level and b) between two specific dates?

Let's start with the UX. I have not thought this all the way out yet, but how about something like:

```shell
--time-start    starting date for triage (e.g. '2020-02-28', '2020-10-01 14:15')
--time-end     ending date for triage range (e.g. '2020-02-28', '2020-10-01 14:15')
```

**Comment by seb128** [comment]
Sorry I forgot to reply to that one, having time-start (and optional end) would work yes, one other option would be to keep the one argument as now and allow it to be a date instead of an integer? (unsure if that would be confusing though?)

**Comment by powersj** [comment]
ok thanks for the feedback! I am leaning towards your suggestion of using a int for the number of days or the date. Then having an option --time-end value.

**Comment by seb128** [comment]
Unsure if the --time-end option is really needed or if assuming it's always 'now' is enough?

**Comment by powersj** [comment]
@seb128  Thanks for the +1 on the review. Please give this a shot and let me know if you run into any issues. I have released it as snap rev 222 (2020.10.08+git3055783)
